### PR TITLE
Add videojs-chromecast@1.1.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by sophie-li**
Pull request for issue: #9825 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kim-company/videojs-chromecast
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 8
   - Count of stars: 71
   - Count of forks: 32
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sophie-li** [comment]
Hmm, I might need more instructions on how to run "npm test" on my local machine. When I go through the root directory `(/ajax/libs/videojs-chromecast)` and run `npm test`, it displays

> videojs-chromecast@1.1.1 test /Users/sophieli/Documents/open-source/cdnjs/ajax/libs/videojs-chromecast
> echo 'Error: no test specified'

> Error: no test specified

**Comment by PeterDaveHello** [comment]
@sophie-li good to see your first contribution here.

Quick question: how do you commit this work? On GitHub web ui or command line?
Ask this because the commits need to be squashed into single one, not sure if you'd like to do it yourself.

**Comment by sophie-li** [comment]
@PeterDaveHello the work is done on command line. i'll handle the squashing

**Comment by PeterDaveHello** [comment]
@sophie-li nice

**Comment by sophie-li** [comment]
@PeterDaveHello just in case, squashed!

**Comment by sophie-li** [comment]
@pvnr0082t Ahh.. I believe the origin files were taken from [videojs-chromecast](https://github.com/kim-company/videojs-chromecast)'s repository and the rest I had to recopy them again on my local machine. Long story short, I tried to run `npm test` with additional files (not uploaded in this PR) on my local machine and the origin files disappeared.

**Comment by pvnr0082t** [comment]
@sophie-li Okay. Try to copy the files from upstream and fix it. :+1:

**Comment by sophie-li** [comment]
@pvnr0082t May I ask how did you find the md5sum identities of the files? I'm currently trying to copy the files from upstream and so far I'm having trouble with it :(

**Comment by PeterDaveHello** [comment]
@sophie-li I think it's via `md5sum` on Linux

**Comment by sophie-li** [comment]
@PeterDaveHello I'm on Mac actually. Still trying to pull back the original files on my local machine..

**Comment by PeterDaveHello** [comment]
@sophie-li “md5” on Mac but I'll respectfully suggest you to use Linux or BSD

**Comment by sophie-li** [comment]
@PeterDaveHello @pvnr0082t I'm blocked on how to retrieve the files with the same md5 as the origins. I even tried creating a new branch based on this one, and the files I have have the same md5  as the non-origins.

**Comment by pvnr0082t** [comment]
@sophie-li I don't get you. Try to remove the file you have and copy the files from upstream again. Thanks.

**Comment by sophie-li** [comment]
@pvnr0082t That's what I tried to do. I removed the files I have locally, that's where I'm stuck: how to get the files from upstream. I only managed to get the same files that are different from the upstreams by doing the following `git checkout my-origin/my-branch -- path/to/file`

**Comment by PeterDaveHello** [comment]
@pvnr0082t I think you need to provide more details/steps about how to fix it, not the very simple concept.

**Comment by pvnr0082t** [comment]
@sophie-li 
Here is the method to manually get the files.
1. clone the `videojs-chromecast` repository using `$ git clone git@github.com:kim-company/videojs-chromecast.git`
2. Get into the directory. `$ cd videojs-chromecast`
3. Check out to the latest version. `$ git checkout v1.1.1`
4. copy the files to under `videojs-chromecast/dist` into `cdnjs/ajax/libs/videojs-chromecast/1.1.1/`.

If there is anything not clear, feel free to ask. :+1:

File: ajax/libs/videojs-chromecast/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kim-company/videojs-chromecast.git"
+  },
+  "author": {
+    "name": "KIM Keep In Mind GmbH, srl",
+    "email": "info@keepinmind.info"
+  },
+  "license": "MIT",
+  "npmName": "videojs-chromecast",
```
Comment (pvnr0082t) [inline ajax/libs/videojs-chromecast/package.json]: @sophie-li Could you please confirm if https://www.npmjs.com/package/videojs-chromecast is officially maintained with the author? If not, we should use git auto-update config instead.

Comment (PeterDaveHello) [inline ajax/libs/videojs-chromecast/package.json]: It seems to for another repo: https://github.com/benjipott/video.js-chromecast

Comment (sophie-li) [inline ajax/libs/videojs-chromecast/package.json]: @pvnr0082t @PeterDaveHello indeed, the repo for this PR is: https://github.com/kim-company/videojs-chromecast.

Comment (PeterDaveHello) [inline ajax/libs/videojs-chromecast/package.json]: So we to fix on this part.

Comment (pvnr0082t) [inline ajax/libs/videojs-chromecast/package.json]: @sophie-li Let's use git auto-update config instead. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)

Comment (sophie-li) [inline ajax/libs/videojs-chromecast/package.json]: @PeterDaveHello hm just to double check, which repo is the correct one? 

1: https://github.com/kim-company/videojs-chromecast 
or
2: https://github.com/benjipott/video.js-chromecast

Comment (PeterDaveHello) [inline ajax/libs/videojs-chromecast/package.json]: As the issue mentions, 1. https://github.com/kim-company/videojs-chromecast

**Pull request merged**